### PR TITLE
Writeups for CyberArc-24

### PR DESCRIPTION
**Pull request by Laav10**

### CODE REVIEW

File: CyberArcCTF-24-Writeups/OSINT/Location Please.../location [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (errorxyz) [inline CyberArcCTF-24-Writeups/OSINT/Location Please.../location]: empty file?

Comment (Laav10) [reply CyberArcCTF-24-Writeups/OSINT/Location Please.../location]: Idk why the files arent uploading

Comment (errorxyz) [reply CyberArcCTF-24-Writeups/OSINT/Location Please.../location]: can you send the files on WA?

File: CyberArcCTF-24-Writeups/Web Exploitation/John Will Triumph/Writeup.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
+# John Will Triumph
+
+**Author: Harsh Iyer**
+**Points: 300**
+**Difficulty: Medium**
+
```
Comment (errorxyz) [inline CyberArcCTF-24-Writeups/Web Exploitation/John Will Triumph/Writeup.md]: Rename file to README.md and fix formatting
https://github.com/Laav10/CTF-Writeups/blob/368c5064f95199fb9251c4aec8087f934addd3a9/CyberArcCTF-24-Writeups/Web%20Exploitation/John%20Will%20Triumph/Writeup.md#john-will-triumph
(Challenge details are displayed on the same line)

File: CyberArcCTF-24-Writeups/Web Exploitation/Rock N Roll/rocknroll.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# Rock N Roll
```
Comment (errorxyz) [inline CyberArcCTF-24-Writeups/Web Exploitation/Rock N Roll/rocknroll.md]: Rename file to README.md

File: CyberArcCTF-24-Writeups/Cryptography/4/README.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+﻿**“4ˮ**
+
```
Comment (errorxyz) [inline CyberArcCTF-24-Writeups/Cryptography/4/README.md:2]: Challenge details missing for "4", "A cipher which begins with V" and "Ezzz!". Tell the author to send a PR for this once merged or feel free to do it if you want.

**Review by errorxyz — CHANGES_REQUESTED** [review]

File: CyberArcCTF-24-Writeups/Web Exploitation/John Will Triumph/Writeup.md (hunk 2 above)
Comment (Laav10) [reply CyberArcCTF-24-Writeups/Web Exploitation/John Will Triumph/Writeup.md]: ok shall fix it

File: CyberArcCTF-24-Writeups/Cryptography/4/README.md (hunk 4 above)
Comment (Laav10) [reply CyberArcCTF-24-Writeups/Cryptography/4/README.md:2]: I noticed it and was planning on asking the author.

Comment (errorxyz) [reply CyberArcCTF-24-Writeups/Cryptography/4/README.md:2]: this is still not resolved

File: CyberArcCTF-24-Writeups/Web Exploitation/Rock N Roll/rocknroll.md (hunk 3 above)
Comment (errorxyz) [reply CyberArcCTF-24-Writeups/Web Exploitation/Rock N Roll/rocknroll.md]: this is still not resolved

**Review by errorxyz — APPROVED** [review]

**Pull request merged**